kgraphql-ktor-stitched/src/main/kotlin/com/apurebase/kgraphql/stitched/schema/execution/RemoteArgumentTransformer.kt

```diff
@@ -0,0 +1,134 @@
+package com.apurebase.kgraphql.stitched.schema.execution
+
+import com.apurebase.kgraphql.Context
+import com.apurebase.kgraphql.request.Variables
+import com.apurebase.kgraphql.schema.execution.ArgumentTransformer
+import com.apurebase.kgraphql.schema.execution.Execution
+import com.apurebase.kgraphql.schema.introspection.TypeKind
+import com.apurebase.kgraphql.schema.introspection.__InputValue
+import com.apurebase.kgraphql.schema.introspection.__Type
+import com.apurebase.kgraphql.schema.model.FunctionWrapper
+import com.apurebase.kgraphql.schema.model.ast.ArgumentNode
+import com.apurebase.kgraphql.schema.model.ast.ArgumentNodes
+import com.apurebase.kgraphql.schema.model.ast.NameNode
+import com.apurebase.kgraphql.schema.model.ast.ValueNode
+import com.apurebase.kgraphql.schema.structure.Field
+import com.apurebase.kgraphql.schema.structure.InputValue
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.BooleanNode
+import com.fasterxml.jackson.databind.node.DoubleNode
+import com.fasterxml.jackson.databind.node.FloatNode
+import com.fasterxml.jackson.databind.node.IntNode
+import com.fasterxml.jackson.databind.node.LongNode
+import com.fasterxml.jackson.databind.node.NullNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.fasterxml.jackson.databind.node.TextNode
+
+class RemoteArgumentTransformer : ArgumentTransformer() {
+    override fun transformArguments(
+        funName: String,
+        receiver: Any?,
+        inputValues: List<InputValue<*>>,
+        args: ArgumentNodes?,
+        variables: Variables,
+        executionNode: Execution,
+        requestContext: Context,
+        field: FunctionWrapper<*>
+    ): List<Any?>? {
+        if (executionNode is Execution.Remote) {
```

Comment:
This can also be an if-expression `return if (...) else (...)`, it would make it clearer for me.